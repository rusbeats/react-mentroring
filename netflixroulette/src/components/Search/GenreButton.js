/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import styled from 'styled-components';

// Homework 8: chose Styled Components
const Button = styled.button`
  color:white;
  background-color: darkgrey;
  font-size:25px;
  margin-left: 5%;
  text-transform: uppercase;
`;

const GenreButton = () => (
  <Button>genre</Button>
);
export default GenreButton;
