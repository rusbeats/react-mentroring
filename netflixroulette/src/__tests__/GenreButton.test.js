import React from 'react';
import { shallow } from 'enzyme';
import GenreButton from '../components/Search/GenreButton';

// Homework 4: Enzyme Snapshot testing
it('GenreButton renders correctly', () => {
  const component = shallow(
    <GenreButton />
  );
  expect(component).toMatchSnapshot();
});
