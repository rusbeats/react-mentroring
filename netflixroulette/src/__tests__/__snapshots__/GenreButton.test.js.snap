// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreButton renders correctly 1`] = `
<styled.button>
  genre
</styled.button>
`;
